crates/bevy_gilrs/src/gilrs_system.rs

```diff
@@ -6,7 +6,7 @@ use gilrs::{EventType, Gilrs};
 
 pub fn gilrs_event_startup_system(gilrs: NonSend<Gilrs>, mut events: EventWriter<GamepadEventRaw>) {
     for (id, _) in gilrs.gamepads() {
-        events.send(GamepadEventRaw(
+        events.send(GamepadEventRaw::new(
```

Comment:
Out of curiosity, why the preference for ::new(...) syntax instead of struct construction pattern?

Reply:
I'm fine with both the `new()` function and struct literals. One of the main reasons I used the `new()` function is because I implemented it either way for users to have an easier time migrating and for API consistency so I just used it as well. For the gamepad types it makes sense that they can be created with a `new` function, because they are relatively small.